samples/Basic/NetCore/Console/SkiaSharpSample.sln

```diff
@@ -1,11 +1,13 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.27130.2010
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.29920.165
 MinimumVisualStudioVersion = 10.0.40219.1
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "SkiaSharpSample", "SkiaSharpSample\SkiaSharpSample.csproj", "{B6D1B569-6ED2-4C9D-9CD3-64ACB89303EA}"
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "SkiaSharp", "..\..\..\..\binding\SkiaSharp\SkiaSharp.csproj", "{ADEB6908-D553-4ED3-94CD-CEC4CF0B34E2}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "SkiaSharpTextOnPath", "SkiaSharpTextOnPath\SkiaSharpTextOnPath.csproj", "{5A6B6815-F1C3-4385-B36D-647EB7F9BD6F}"
```

Comment:
Is it possible to add a gallery sample instead? If I did it right, it should just be a matter of adding a new file here: https://github.com/mono/SkiaSharp/tree/master/samples/Gallery/Shared/Samples

Reply:
Of course, I missed that :)